frontend/app-development/layout/PageHeader/SubHeader/SettingsModalButton/SettingsModal/components/Tabs/Maskinporten/ScopeList/ScopeList.tsx

```diff
@@ -1,20 +1,36 @@
-import React, { type ReactElement } from 'react';
+import React, { useState, type ChangeEvent, type ReactElement } from 'react';
 import classes from './ScopeList.module.css';
-import { StudioAlert, StudioSpinner } from '@studio/components';
+import {
+  StudioAlert,
+  StudioCheckboxTable,
+  type StudioCheckboxTableRowElement,
+  StudioHeading,
+  StudioParagraph,
+  StudioSpinner,
+} from '@studio/components';
 import { useGetScopesQuery } from 'app-development/hooks/queries/useGetScopesQuery';
 import { useTranslation } from 'react-i18next';
+import { type MaskinportenScope } from 'app-shared/types/MaskinportenScope';
+import { useGetSelectedScopesQuery } from 'app-development/hooks/queries/useGetSelectedScopesQuery';
+import { mapRowElementsToSelectedScopes, mapScopesToRowElements } from './utils';
 
 export const ScopeList = (): ReactElement => {
   const { t } = useTranslation();
   const { data: scopes, isPending: isPendingScopes } = useGetScopesQuery();
+  const { data: selectedScopes, isPending: isPendingSelectedScopes } = useGetSelectedScopesQuery();
+
+  console.log('scopes', scopes);
+  console.log('selectedScopes', selectedScopes);
+
   const hasScopes: boolean = scopes?.length > 0;
+  const isPendingQueries: boolean = isPendingScopes || isPendingSelectedScopes;
 
-  if (isPendingScopes) {
+  if (isPendingQueries) {
     return <StudioSpinner spinnerTitle={t('general.loading')} />;
   }
 
   if (hasScopes) {
-    return <div>List of scopes and possibility to select scope comes here</div>;
+    return <ScopeListContent allScopes={scopes} selectedScopes={selectedScopes} />;
```

Comment:
We should display scopes from our database, even if no scopes exist in Maskinporten. Based on my conversation with Anniken, checkboxes for scopes present in our database but not in Maskinporten will be disabled in the UI. Users should only be able to modify scopes they can access within Maskinporten.